client/my-sites/marketing/tools/index.tsx

```diff
@@ -93,7 +93,7 @@ export const MarketingTools: FunctionComponent = () => {
 					description={ translate(
 						"Hire our dedicated experts to build a handcrafted, personalized website. Share some details about what you're looking for, and we'll make it happen."
 					) }
-					imagePath={ builtByWp }
+					imagePath={ wordPressLogo }
 					imageAlt={ translate( 'Built by WordPress.com logo' ) }
```

Comment:
Might be worth updating this alt text accordingly? But since it's purely decoration, I'm not even sure if it's necessarily to include any alt text at all.